src/index.ts

```diff
@@ -21,6 +21,7 @@ type Options<T = TypeNode> = {
     currentType: T;
     customScalars?: ScalarMap;
     transformUnderscore: boolean;
+    dynamic?: boolean;
```

Comment:
could you name this `dynamicValues` maybe?

Reply:
Sure!